src/module/canvas/token/aura/map.ts

```diff
@@ -53,7 +53,9 @@ export class AuraRenderers extends Map<string, AuraRenderer> {
                 : this.token.combatant?.encounter.active);
 
         return (
-            canvas.scene?.grid.type === CONST.GRID_TYPES.SQUARE &&
+            canvas.scene !== null &&
+            // Check if aura is supported
+            canvas.scene.canHaveAuras &&
```

Comment:
```suggestion
            canvas.scene?.canHaveAuras &&
```

Reply:
Type 'boolean | undefined' is not assignable to type 'boolean'.
  Type 'undefined' is not assignable to type 'boolean'.ts(2322)

Reply:
Dammit. `!!canvas.scene?.canHaveAuras &&` is a bit heinous here too. I guess just the comment change then.

Reply:
Actually, just remove the comment, its self commenting

Reply:
okay sounds good. I mean I love a good heinous one liner but it's def not an approachable piece of code.